docs/source/augly.audio.rst

```diff
@@ -0,0 +1,53 @@
+augly.audio package
+===================
+
+Submodules
+----------
+
+augly.audio.composition module
+------------------------------
+
+.. automodule:: augly.audio.composition
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+augly.audio.functional module
+-----------------------------
+
+.. automodule:: augly.audio.functional
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+augly.audio.intensity module
+----------------------------
+
+.. automodule:: augly.audio.intensity
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+augly.audio.transforms module
+-----------------------------
+
+.. automodule:: augly.audio.transforms
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+augly.audio.utils module
+------------------------
+
+.. automodule:: augly.audio.utils
+   :members:
+   :undoc-members:
+   :show-inheritance:
```

Comment:
i don't think it's needed to publicize the `utils` module, since that's meant for internal development as opposed to external use. tagging @zpapakipos to ensure she agrees